cli/functionary/environment.py

```diff
@@ -1,39 +1,21 @@
-import json
-
 import click
-import requests
 
+from .client import get
 from .config import get_config_value, save_config_value
 
 
 def _get_environment_list():
     """
     Helper function to get the environment list from host
     """
-    token = get_config_value("token")
-    teams_url = get_config_value("host") + "/api/v1/teams"
-    header = {"Authorization": f"Token {token}"}
-    try:
-        response = requests.get(teams_url, headers=header)
-    except requests.ConnectionError:
-        raise click.ClickException("Could not connect to host")
-    except requests.Timeout:
-        raise click.ClickException("Timeout occured while trying to connect to host")
-
-    if response.ok:
-        data = json.loads(response.text).get("results")
+    data = get("teams")
```

Comment:
You may as well just call this `teams` rather than `data`.  In this particular case we never get too far away from this declaration that one would lose track of what this is, but as a general practice you want to have descriptive names for things so that someone reading a bit of code that uses this would still have some clue what it is even removed from the context of this assignment.